resources/asciidoctor/spec/edit_me_spec.rb

```diff
@@ -13,17 +14,41 @@
     Asciidoctor::Extensions.unregister_all
   end
 
+  spec_dir = File.dirname(__FILE__)
+
+  it "has a nice error message if you are missing the edit url" do
```

Comment:
I love test names like this. :+1:

Totally slays `test_error_message`.

Reply:
I like this about rspec a lot! "Ruby as a DSL" can have all kinds of problems with error reporting and stuff, but here it is super nice!